plugins/bitbucket-cloud-common/scripts/prepare-schema.js

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const SCHEMA_SOURCE =
+  'https://developer.atlassian.com/cloud/bitbucket/swagger.v3.json';
+
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+const destFile = `${__dirname}/../bitbucket-cloud.oas.json`;
```

Comment:
So this file isn't version controlled - should it be? This is all so much and so magic, that it's a bit of a head scratcher :) I Was kinda hoping that the original swagger schema would have been directly useful. And if it isn't, for some reason, we'd be able to script a tweak (kinda as you do here) that updates a local version controlled tweaked copy of it. Does that make sense?

Reply:
yes, we can version control it. At first I had it included, but than dropped it. I can add it back again if you prefer that.